web/components/bet/limit-slider.tsx

```diff
@@ -0,0 +1,143 @@
+import clsx from 'clsx'
+import {
+  CPMMBinaryContract,
+  CPMMMultiContract,
+  PseudoNumericContract,
+  StonkContract,
+} from 'common/contract'
+import Slider from 'rc-slider'
+import { useEffect, useRef, useState } from 'react'
+import { useWindowSize } from 'web/hooks/use-window-size'
+import { Col } from '../layout/col'
+import { Row } from '../layout/row'
+
+export function LimitSlider(props: {
+  lowLimitProb: number
+  setLowLimitProb: (prob: number) => void
+  highLimitProb: number
+  setHighLimitProb: (prob: number) => void
+  mobileView?: boolean
+}) {
+  const {
+    lowLimitProb,
+    setLowLimitProb,
+    highLimitProb,
+    setHighLimitProb,
+    mobileView,
+  } = props
+
+  const mark = (value: number) => (
+    <span className="text-ink-400 text-xs">
+      <div className={'sm:h-0.5'} />
+      {value}%
+    </span>
+  )
+
+  const targetRef = useRef<HTMLDivElement>(null)
+  const [sliderWidth, setSliderWidth] = useState(0)
+  const window = useWindowSize()
+  useEffect(() => {
+    if (targetRef.current) {
+      setSliderWidth(targetRef.current.offsetWidth)
+    }
+  }, [window.width])
+
+  const lowLimitPosition = sliderWidth * ((lowLimitProb ?? 0) / 100)
+  const highLimitPosition = sliderWidth * ((highLimitProb ?? 0) / 100)
+  const LIMIT_LABEL_MIN_DISTANCE = 130
+  return (
+    <Col className="mb-8">
+      <Row className="mt-1 mb-4 gap-4">
+        <Col className="w-full">
+          <div className="text-ink-800 text-sm">
+            Attempt to keep between probabilities:
```

Comment:
Not bad, though let me see what I can think of.

"Bet when the probability crosses a lower or upper bound"
"Bet if the probability leaves the range"
"Bet at a lower or upper bound"